src/Parser.zig

```diff
@@ -2076,6 +2087,40 @@ const Enumerator = struct {
         e.res = res;
         // TODO adjust res type to try to fit with the previous type
     }
+
+    fn tagSpec(e: *const Enumerator, p: *Parser, is_packed: bool, tok: TokenIndex) !Type.Specifier {
```

Comment:
The `spec` in `enumSpec` and `recordSpec` comes from the official grammar, maybe it would be better to call this `getTagType` or even just `tagType`?